bin/i18n/sync-down.rb

```diff
@@ -21,13 +21,16 @@ def sync_down
     # We really only care about general progress monitoring, so we remove or
     # ignore any things we identify as "noise" in the output.
     Open3.popen2(command) do |_stdin, stdout, status_thread|
-      stdout.each_line do |line|
+      while line = stdout.gets
         # strip out the progress spinner, which is implemented as the sequence
         # \-/| followed by a backspace character
         line.gsub!(/[\|\/\-\\][\b]/, '')
 
         # skip lines detailing individual file extraction
-        next if line.start_with?("Extracting: ", "Building ")
+        next if line.start_with?("Extracting: ")
```

Comment:
I don't have a strong opinion but I'm curious how useful the `Building..` lines actually are. Happy to add it back in and see how noisy the sync is though

Reply:
I mostly want some way to see the progress of the sync down as it happens, since it's by far the longest-running step of the sync. I'd definitely be down to replace these lines with a less-fine-grained progress report, but I'd rather have them than nothing.